runtests.sh

```diff
@@ -46,6 +46,7 @@ python3 -m coverage 1>/dev/null 2>&1 && coverage="true"
 run_static_tests(){
     SRC_PATHS="bin snapcraft snaps_tests external_snaps_tests setup.py"
     python3 -m flake8 --max-complexity=10 $SRC_PATHS
+    mypy --ignore-missing-imports --follow-imports=silent -p snapcraft
```

Comment:
Last question: we go to lengths to check more than just `snapcraft` in the flake8 tests. Is there a reason we're not doing the same for mypy?

Reply:
I gave it thought and there is minimal to no gain in checking setup.py while the others are in there path to going away so not worth fixing if they have problems.